src/main/java/com/shapesecurity/shift/es2016/codegen/CodeGen.java

```diff
@@ -50,6 +50,14 @@ public static String codeGen(@Nonnull Module module) {
         return codeGen(module, COMPACT);
     }
 
+    @Nonnull
+    public static String codeGen(@Nonnull Program program) {
```

Comment:
Let's remove the Script/Module cases above and replace these calls with their implementation.

Reply:
Also, let's do the same for the one that takes a `pretty` parameter.